pinot-tools/src/main/resources/examples/stream/meetupRsvp/complexTypeHandling_meetupRsvp_schema.json

```diff
@@ -0,0 +1,40 @@
+{
+  "schemaName": "meetupRsvp",
```

Comment:
Can we add one test where the the user chooses to use collection unnesting rule flattening everything thereby resulting in every leaf node as a primitive column in the schema ? Would be great to add a test with reasonably complex hierarchy of source data especially including list inside a list

Reply:
Yes, we can do it for this meetupRsvp. Also, the unit test has a case of unnesting two sibiling collections.